docs/data/data-grid/column-definition/column-definition.md

```diff
@@ -71,9 +71,19 @@ In the following demo, a formatter is used to display the tax rate's decimal val
 
 The value generated is only used for rendering purposes.
 Filtering and sorting do not rely on the formatted value.
-Use the [`valueParser`](/x/react-data-grid/cells/#value-parser) to support filtering.
+Use the [`valueParser`](#value-parser) to support filtering.
 
-## Rendering content
+### Value parser
```

Comment:
A section that we seems to have forgotten

Reply:
I forgot to remove the demo files 
The section is now [here](https://mui.com/x/react-data-grid/editing/#value-parser-and-value-setter)

See https://github.com/mui/mui-x/pull/4600#discussion_r856245602

Reply:
But this section only cover row editing, isn't the value parser also used for filtering?